docs/receivers.md

```diff
@@ -4,69 +4,69 @@ Below is a list of supported receivers with links to their documentation pages.
+| Fluentd Forward Receiver                    | [fluentforwardreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.77.0/receiver/fluentforwardreceiver/README.md) |
+| Google Cloud Pub/Sub Receiver               | [googlecloudpubsubreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.77.0/receiver/googlecloudpubsubreceiver/README.md) |
+| Google Cloud Spanner Receiver               | [googlecloudspannerreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.77.0/receiver/googlecloudspannerreceiver/README.md) |
+| Host Metrics Receiver                       | [hostmetricsreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.77.0/receiver/hostmetricsreceiver/README.md) |
+| InfluxDB Receiver                           | [influxdbreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.77.0/receiver/influxdbreceiver/README.md) |
+| Jaeger Receiver                             | [jaegerreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.77.0/receiver/jaegerreceiver/README.md) |
+| JMX Receiver                                | [jmxreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.77.0/receiver/jmxreceiver/README.md) |
+| journald Receiver                           | [journaldreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.77.0/receiver/journaldreceiver/README.md) |
+| Kubernetes Cluster Receiver                 | [k8sclusterreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.77.0/receiver/k8sclusterreceiver/README.md) |
+| Kubernetes Events Receiver                  | [k8seventsreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.77.0/receiver/k8seventsreceiver/README.md) |
+| Kubernetes kubelet Stats Receiver           | [kubeletstatsreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.77.0/receiver/kubeletstatsreceiver/README.md) |
+| Kafka Receiver                              | [kafkareceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.77.0/receiver/kafkareceiver/README.md) |
+| Kafka Metrics Receiver                      | [kafkametricsreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.77.0/receiver/kafkametricsreceiver/README.md) |
+| Memcached Receiver                          | [memcachedreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.77.0/receiver/memcachedreceiver/README.md) |
+| Microsoft .NET Process Diagnostics Receiver | [dotnetdiagnosticsreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.77.0/receiver/dotnetdiagnosticsreceiver/README.md) |
```

Comment:
I think we need to remove this documentation.

Reply:
good call, removed